README.md

```diff
@@ -69,32 +69,36 @@ Also, you can of course not use some features outlined here. That's why the whol
    To do so just run `gpg --verify download.zip.asc`.
 3. Make sure that all script are not writable by the backup user, if it is different than root.
 
-### 2. Setup statistics (optional)
+### 2. Enter the backup parameters
 
-In order to use the logging and reporting functionality, you have to create some dirs. They all have to be writable by the user running the scripts.
+In the ['config directory'](config/) you will find an example configuration file. Set the variables according to your already initialized borg backup. Each backup has its own configuration file.
+
+### 3. Setup statistics (optional)
+
+To use the logging and reporting functionality, you have to create some dirs. These have to be writable by the user running the scripts.
 
 1. For logging: Create `/var/log/borg` with appropiate permissions.
 2. Create a subdirectry called `/var/log/borg/last` (configurable as [`LAST_BACKUP_DIR`](borgcron.sh#L9)). There the `.time` files will be written to containing the tiome of the last backup execution.
-3. Include/add the [`tools/checklastbackup.sh`](tools/checklastbackup.sh) script to your `~/.bashrc`, `~/.zshrc` or similar depending on your shell). It will use the `.time` files to check the latest execution of the backups. You may also configure it's [`CRITICAL_TIME`](tools/checklastbackup.sh#L8), which defines the time the backup notifies you of a problem.
+3. Include/add the [`tools/checklastbackup.sh`](tools/checklastbackup.sh) script to your `~/.bashrc`, `~/.zshrc` or similar, depending on your shell). It will read the `.time` files to display the time of  the last execution of your backups, when you login into your shell. You may also set a period of time ([`CRITICAL_TIME`](tools/checklastbackup.sh#L8)) in order to get a notification, if no successful backup has been made within that time.
```

Comment:
Please make clear the "period of time" (maybe [time period](https://dict.leo.org/englisch-deutsch/time+period) is better) is connected/inside of the login script you mention in the previous sentence. Currently it may look like this period is not related to the script.
Simply change "set a period" to "set the period" e.g.